src/customer.spec.ts

```diff
@@ -32,11 +33,11 @@ describe("querier", () => {
   it("parses query results by default", async () => {
     const customer = newCustomer({});
     mockPaginatedSearch(customer);
-    const mockedParse = mockParse(mockParsedValues);
+    mockParse(mockParsedValues);
+
     // @ts-expect-error private method
     const { response } = await customer.querier(mockGaqlQuery);
 
-    expect(mockedParse).toHaveBeenCalled();
```

Comment:
Not quite -- since the parser is now called from within `paginatedSearch()`, which is replaced by a mock in this test, checking that the parser has been called doesn't make sense. It would be testing that the mocked `paginatedSearch()` calls parse()... which it does, but at that point I feel that the tests are getting a bit too meta haha